### DISCUSSION TITLE
Welcome to nixos-and-flakes-book Discussions!

### DISCUSSION THREAD

**Discussion by ryan4yin** [ryan4yin/nixos-and-flakes-book#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ollekv** [comment]
Just want to send a thanks for this repo/book. Just when I was to give up understanding flake,and NixOS overall, I see a light in the tunnel following your guidance. Still far away but at least light 😃
